### PR TITLE
Improved Redirecting paragraph of Testing page

### PR DESCRIPTION
**Pull request by FrancescoBorzi**

### CODE REVIEW

File: testing.rst [hunk 1]
```diff
@@ -505,7 +505,7 @@ afterwards with the ``followRedirect()`` method::
     $crawler = $client->followRedirect();
 
 If you want the client to automatically follow all redirects, you can
-force him with the ``followRedirects()`` method::
+force him by calling the ``followRedirects()`` method before performing the request::
```
Comment (wouterj) [inline testing.rst]: Love your addition! If you have the time, please make one more minor change in this change: Change "him" to "them". We want to be as gender-neutral as possible in the docs :)

**Comment by wouterj** [comment]
👍 
Status: reviewed

File: testing.rst [hunk 2]
```diff
@@ -505,7 +505,7 @@ afterwards with the ``followRedirect()`` method::
     $crawler = $client->followRedirect();
 
 If you want the client to automatically follow all redirects, you can
```
Comment (javiereguiluz) [inline testing.rst]: @ShinDarth thanks for this improvement. I like it ... but I still think that the original phrase is not very good. What if we reword it as follows:

```
You can make the client to automatically follow all redirects by
calling the ``followRedirects()`` method before making the first request::
```

File: testing.rst (hunk 1 above)
Comment (ShinDarth) [inline testing.rst]: right, done!

**Comment by xabbuh** [comment]
Thank you @ShinDarth.

**Pull request merged**